modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptClientCodegen.java

```diff
@@ -96,6 +82,8 @@ public JavascriptClientCodegen() {
                 "version of the project (Default: using info.version or \"1.0.0\")"));
         cliOptions.add(new CliOption(PROJECT_LICENSE_NAME,
                 "name of the license the project uses (Default: using info.license.name)"));
+        cliOptions.add(new CliOption(USE_PROMISES,
+                "use Promises as return values from the client API, instead of superagent callbacks (Default: false)"));
```

Comment:
Instead of putting the default value in the description, you can use `.defaultValue` instead. Here is an example: https://github.com/swagger-api/swagger-codegen/blob/ee8547f3b4e6260c6b9ae3dbfd94652109adcb3e/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PerlClientCodegen.java#L79